### DISCUSSION TITLE
How to config dynamic function on server/api ?

### DISCUSSION THREAD

**Discussion by little3201** [nuxt/nuxt#18207] [Questions]
if i want to $fetch("/api/posts") work, i should to create a file named posts.js/posts.ts under server/api directory,

now, if i want to $fetch("/api/posts") and $("/api/posts/xxx") work, how i can do?

like: 

list data from '/api/posts', and single data from '/api/posts/124' or next data from '/api/posts/124/next'

**Comment by Norel** [comment]
Not supported yet by h3
See here https://github.com/nuxt/framework/discussions/1137 for a possible workaround
